astrology.lic

```diff
@@ -28,27 +28,37 @@ class Astrology
     @ap_source = settings.astral_plane_training['train_source']
     @ap_destination = settings.astral_plane_training['train_destination']
     @predict_event = settings.predict_event
-	@full_pools = settings.astrology_use_full_pools
+    @full_pools = settings.astrology_use_full_pools
```

Comment:
we could avoid duplicating the `target = @full_pools ? 10 : 1` logic by storing the calculated target value into something like `@prediction_pool_target` (and then not storing `@full_pools`).